docs/install_guides/installing-openvino-yum.md

```diff
@@ -1,10 +1,210 @@
+
+      sudo yum install openvino-2022.3.0
+
+@endsphinxdirective
+
+
+#### Check for Installed Packages and Version
+
+Run the following command:
+
+@sphinxdirective
+
+   .. code-block:: sh
+
+      yum list installed 'openvino*'
+
+@endsphinxdirective
+
+
+#### Uninstall OpenVINO Runtime
+
+@sphinxdirective
+
+.. tab:: The Latest Version
+
+   Run the following command:
+
+   .. code-block:: sh
+
+      sudo yum autoremove openvino
+
+
+.. tab::  A Specific Version
+
+   Run the following command:
+
+   .. code-block:: sh
+
+      sudo yum autoremove openvino-<VERSION>.<UPDATE>.<PATCH>
+
+   For example:
+
+   .. code-block:: sh
+
+      sudo yum autoremove openvino-2022.3.0
+
+@endsphinxdirective
+
+
+### Step 3 (Optional): Install OpenCV from YUM
+
+OpenCV is necessary to run C++ demos from Open Model Zoo. Some OpenVINO samples can also extend their capabilities when compiled with OpenCV as a dependency. OpenVINO provides a package to install OpenCV from YUM:
+
+@sphinxdirective
+
+.. tab:: The Latest Version
+
+   Run the following command:
+
+   .. code-block:: sh
+
+      sudo yum install openvino-opencv
+
+.. tab::  A Specific Version
+
+   Run the following command:
+
+   .. code-block:: sh
+
+      sudo yum install openvino-opencv-<VERSION>.<UPDATE>.<PATCH>
+
+@endsphinxdirective
+
+
+### Step 4 (Optional): Install Software Dependencies
+
+After you have installed OpenVINO Runtime, if you decided to [install OpenVINO Model Development Tools](installing-model-dev-tools.md), make sure that you install external software dependencies first.
+
+Refer to <a href="openvino_docs_install_guides_installing_openvino_linux.html#install-external-dependencies">Install External Software Dependencies</a> for detailed steps.
+
+### Step 5 (Optional): Configure Inference on Non-CPU Devices
+
+@sphinxdirective
+
+.. tab:: GNA
+
+   To enable the toolkit components to use Intel® Gaussian & Neural Accelerator (GNA) on your system, follow the steps in :ref:`GNA Setup Guide <gna guide>`.
+
+.. tab:: GPU
+
+   To enable the toolkit components to use processor graphics (GPU) on your system, follow the steps in :ref:`GPU Setup Guide <gpu guide>`.
+
+.. tab:: NCS 2
+
+   To perform inference on Intel® Neural Compute Stick 2 powered by the Intel® Movidius™ Myriad™ X VPU, follow the steps on :ref:`NCS2 Setup Guide <ncs guide>`.
+   <!--For more details, see the `Get Started page for Intel® Neural Compute Stick 2 <https://software.intel.com/en-us/neural-compute-stick/get-started>`.-->
+
+.. tab:: VPU
+
+   To install and configure your Intel® Vision Accelerator Design with Intel® Movidius™ VPUs, see the :ref:`VPU Configuration Guide <vpu guide>`.
+   After configuration is done, you are ready to run the verification scripts with the HDDL Plugin for your Intel® Vision Accelerator Design with Intel® Movidius™ VPUs.
+
+   .. warning::
+      While working with either HDDL or NCS, choose one of them as they cannot run simultaneously on the same machine.
```

Comment:
remove VPU, NSC2, GNA plugin
they are not available in RPM

Reply:
Have removed, commit underway.